docs/modules/cli/pages/install.adoc

```diff
@@ -0,0 +1,36 @@
+= Install guide
+
+Assumptions:
+
+* [x] You've installed Node 8.11+ on your machine.
+
+On this page, you'll learn:
+
+* [x] How to install Asciidoctor.js CLI.
+
+== Install Asciidoctor.js CLI
+
+You must have Asciidoctor.js installed to use it.
+
+Install Asciidoctor.js and the CLI globally and you'll have access to the `asciidoctorjs` command anywhere on your system.
+Open a terminal and type:
+
+[source]
+$ npm i -g asciidoctor.js asciidoctor-cli
```

Comment:
Generally I prefer commands to use the following syntax (offset by one space):

```
 $ npm i -g asciidoctor.js asciidoctor-cli
```

Then we can use an extension to beauty them up.